redis/src/aio/connection_manager.rs

```diff
@@ -18,6 +18,82 @@ use std::sync::Arc;
 use tokio_retry::strategy::{jitter, ExponentialBackoff};
 use tokio_retry::Retry;
 
+/// Apply a maximum delay. No retry delay will be longer than this `max_delay`.
+#[derive(Clone, Debug, Default)]
+pub struct ConnectionManagerConfig {
+    /// The resulting duration is calculated by taking the base to the `n`-th power,
+    /// where `n` denotes the number of past attempts.
+    exponent_base: u64,
+    /// A multiplicative factor that will be applied to the retry delay.
+    ///
+    /// For example, using a factor of `1000` will make each delay in units of seconds.
+    factor: u64,
+    /// number_of_retries times, with an exponentially increasing delay
+    number_of_retries: usize,
+    /// Apply a maximum delay. No retry delay will be longer than this 'duration' milliseconds.
```

Comment:
Apply a maximum delay between connection attempts. The delay between attempts won't be longer than `max_delay` milliseconds.